sopel/irc/backends.py

```diff
@@ -101,7 +101,7 @@ def irc_send(self, data):
 
         :param bytes data: raw line to send
 
-        This uses :meth:`asynchat.async_chat.send` method to send ``data``
+        This uses :meth:`asyncore.dispatcher.send` method to send ``data``
```

Comment:
This is an interesting thing to catch now. I'd say this should be in #1890, but I merged it earlier… whoops! 😹

Reply:
ikr